pyms/flask/services/metrics.py

```diff
@@ -47,14 +46,21 @@ class Service(DriverService):
     """
     Adds [Prometheus](https://prometheus.io/) metrics using the [Prometheus Client Library](https://github.com/prometheus/client_python).
     """
-    config_resource: Text = "metrics"
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.metrics_blueprint = Blueprint("metrics", __name__)
         self.init_registry()
         self.serve_metrics()
 
+    def init_action(self, microservice_instance):
```

Comment:
I do like this! If I do understand this correctly, with this all the logic is on the service and you don't have to touch the `create_app` file. So, you could delete https://github.com/python-microservices/pyms/blob/master/pyms/flask/app/create_app.py#L188?

Reply:
I agree. Really nice change. First step for eventual 3rd party drivers too.